ui/desktop/app/routes/scopes/scope/projects/settings/index.js

```diff
@@ -4,5 +4,21 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class ScopesScopeProjectsSettingsIndexRoute extends Route {}
+export default class ScopesScopeProjectsSettingsIndexRoute extends Route {
+  @service ipc;
+
+  async model() {
+    const { versionNumber: cliVersion } =
+      await this.ipc.invoke('getCliVersion');
+    const getCacheDaemon = await this.ipc.invoke('getCacheDaemonStatus');
+    const { version } = getCacheDaemon;
+    // Format the version
+
+    const formattedCacheVersion = version && /(\d+\.\d+\.\d+)/.exec(version)[0];
```

Comment:
you are right, I did not need that. I was using that in case I did not have a version, but now modified the model to handle the case so it is not needed. thanks for the catch